src/app/courses/courses.component.ts

```diff
@@ -214,12 +216,20 @@ export class CoursesComponent implements OnInit, AfterViewInit, OnDestroy {
     this.parent ? this.router.navigate([ '/manager' ]) : this.router.navigate([ '/' ]);
   }
 
-  enrollInSelected(courseIds, type) {
+  enrollLeaveToggle(courseIds, type) {
     this.coursesService.courseAdmissionMany(courseIds, type).subscribe((res) => {
       this.selectedNotEnrolled = 0;
+      this.selectedNotLeft = 0;
     }, (error) => ((error)));
   }
 
+  courseToggle(courseId, type) {
```

Comment:
Please don't move functions around.  It makes it more difficult to see the actual changes to the function.